modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To access the Thanos Querier API endpoints.
+* To manage alert silences.
+* To monitor the Alertmanager API.
+
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
```

Comment:
(discussion) Now that we have a split between role and clusterrole in the other table, should we do the same here?

Reply:
I did not want to do this here, because we only list one cluster role and implementing a table with just one role is not ideal :sweat_smile: . So I think having it in one table (at least for now) is a better choice. :+1: